src/hotspot/share/classfile/classFileParser.cpp

```diff
@@ -1194,11 +1204,21 @@ static void parse_annotations(const ConstantPool* const cp,
         && s_tag_val == *(abase + tag_off)
```

Comment:
I see in previous lines that the count restriction on `@Contended` remains, and such a restriction is absent for `@Deprecated`. Intentional?

Reply:
For `@Deprecated` it previously asserted if there were <= 2 members and we needed to remove that. By examining only the first two members we are implicitly allowing for there to be more, but we will ignore all members if the unexpected ones come first. For `@Contended` we simply ignore the "value" setting if there is more than one member. I guess I could check for `count >= 1` to allow the same level of leniency.

Reply:
I meant > 2 of course.

Reply:
But again the intent here is not to allow fabricated annotations to be treated like the real ones, but just to ensure such fabricated annotations don't cause a crash, or other failure. So I will keep the count check for exactly 1.